docs/static/settings-file.asciidoc

```diff
@@ -114,6 +114,22 @@ The `logstash.yml` file includes the following settings.
 | (Beta) Load Java plugins in independent classloaders to isolate their dependencies.
 | `false`
 
+| `pipeline.ordered`
+a|
+Valid options are:
+
+* `auto`
+* `true`
+* `false`
+
+Set the pipeline event ordering.
+`auto` will  automatically enable ordering if the `pipeline.workers` setting is also set to `1`.
+`true` will enforce ordering on the pipeline and prevent logstash from starting
+if there are multiple workers.
+`false` will disable any extra processing necessary for preserving ordering.
```

Comment:
This statement seems a bit ambiguous.   

Does it mean something like, "will disable any extra processing so that ordering is preserved." ?

Reply:
hmm, not sure how to formulate it but what I mean here is that by setting it to `false`, ordering will not be guaranteed and by doing so you will not be penalized by the processing cost of preserving order. Maybe this is not the right place to mention this?

Reply:
Thanks for clarifying.  That makes so much more sense.  And I think this IS a good place to mention this. 

What about something like: 
"`false` will disable the processing required to preserve order. Ordering will not be
guaranteed, but you save the processing cost of preserving order. "

Reply:
adjusted!